kaldb/src/test/java/com/slack/kaldb/chunkManager/IndexingChunkManagerTest.java

```diff
@@ -532,6 +532,8 @@ public void testAddAndSearchMessageInSpecificChunks() throws Exception {
         2);
   }
 
+  // TODO: Add a unit test when message ingestion fails.
+
   @Test
   public void testAddMessageWithPropertyTypeErrors() throws Exception {
```

Comment:
This probably needs renamed, as it no longer a property type error - it silently handles this and converts the field.

Reply:
Renamed.

Reply:
Doesn't look like this was addressed

Reply:
Fixed now.